bld/CMakeLists.txt

```diff
@@ -66,11 +66,13 @@ endif()
 
 if (CMAKE_VERSION VERSION_LESS "3.1")
```

Comment:
Minimum required I think is 2.8.11, so if someone has that installed, we need to set the std differently than post ver 3.1, which is when they added the STD variables.  

I am getting curious, 4.8.x works fine (used by travis), I might set up Debian Docker image now and try this.